src/extension/android/adb.ts

```diff
@@ -3,8 +3,12 @@
 
 import * as Q from "q";
 
-import {ChildProcess} from "../../common/node/childProcess";
+import { ChildProcess, ISpawnResult } from "../../common/node/childProcess";
 import {CommandExecutor} from "../../common/commandExecutor";
+import * as path from "path";
+import { FileSystem } from "../../common/node/fileSystem";
+import { ILogger } from "../log/LogHelper";
+const fs = new FileSystem();
```

Comment:
Why we can't use `import * as fs from "fs"` here?

Reply:
@iTOYS Because we need some methods not from core `fs` module, but from util module `fileSystem`.

Reply:
@iTOYS 
https://github.com/Microsoft/vscode-react-native/pull/754/files/85c625d30da82ad693f52a9687be87615ca5154e#diff-0d185dabb7af8a387b988133ca5ca4d8R184
https://github.com/Microsoft/vscode-react-native/pull/754/files/85c625d30da82ad693f52a9687be87615ca5154e#diff-0d185dabb7af8a387b988133ca5ca4d8R177